package.json

```diff
@@ -95,6 +96,9 @@
     },
     "localforage": {
       "optional": true
+    },
+    "localforage-removeitems": {
+      "optional": true
```

Comment:
this peer dependency is optional, similar to `localforage`. Reference: https://github.com/Expensify/react-native-onyx/pull/132/files#r873998291